src/api/api.ts

```diff
@@ -1,42 +1,7 @@
-const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
 
-export const getRequest = async (endpoint: string) => {
-  try {
-    const response = await fetch(`${BASE_URL}${endpoint}`);
+export async function getAllUsers() {
```

Comment:
Adding types to the functions' parameters and return types can improve code readability and help catch errors during development.